nilearn/plotting/img_plotting.py

```diff
@@ -954,13 +954,13 @@ def plot_roi(
         contours.
 
     %(cmap_lut)s
-        Default=`gist_ncar`.
+        Default = `gist_ncar`.
```

Comment:
```suggestion
        Default=`gist_ncar`.
```(for consistencey, I'm not 100%sure that this is the right way to display it)

Reply:
I don't think we have brought our nitpicking to that level just yet 😉